### ISSUE TITLE
Some languages are missing from ESpeak NG in NVDA next-14373,6bbe5915

### ISSUE THREAD

**Issue by hkatic** [nvaccess/nvda#7557]
### Steps to reproduce:
> 1. Switch to ESpeak NG.
> 2. Open Voice Settings.
> 3. Try to switch to languages such as Croatian, French, Italian, Polish, Russian, Hungarian, Finnish, Czech, Slovak, Slovenian or Spanish.

### Expected behavior:
> All ESpeak NG languages should be available in a list.

### Actual behavior:
> Those languages mentioned above plus may be some more are missing in ESpeak NG starting from this snapshot.

### System configuration:
NVDA version:
> next-14373,6bbe5915

NVDA Installed or portable: Installed

Windows version:
> Windows 10 Version 1703 (OS Build 16251.1002)

Please note that restarting NVDA doesn't fix the problem.

**Comment by ehollig** [comment]
I am not able to replicate this. Using a portable version of NVDA and opening the voice dialog, I am able to locate Croatian, French, Italian, Polish, Russian, Hungarian, Finnish, Czech, Slovak, Slovenian and Spanish in the list. Have you tried reinstalling NVDA? Alternatively, has something changed in the synthDrivers folder of NVDA config?

**Comment by hkatic** [comment]
Hello,
So, the problem wasn't with configuration, because resetting the configuration didn't fix the problem.
The problem was, in fact, with the installation (probably some files were not copied properly during the automatic update process).
So, I got all ESpeak voices back when I've reinstalled the latest Next snapshot.
We can close this issue, but validating the update after downloading and before installing it may be something to consider for the future, to avoid situations like this where some files may be missing after the update.

**Comment by ehollig** [comment]
Thank you @hkatic for the response. Closing as worksforme. CC @michaelDCurran for the validating the update after downloading and before installing idea expressed in https://github.com/nvaccess/nvda/issues/7557#issuecomment-326788505. This idea may need to be expressed in a new issue/suggestion.
